File: test iphone.c
#include <stdio.h>

int main()
{
	printf("Hello there! I wrote this on an iPhone in Linux");
	printf("Whats your name? ");
	char name[100] ={"0"};
	scanf("%s", name);
	printf("And your age is? ");
	int age = 0;
	scanf("%d", &age);
	printf("Hello %s!\nYour age is %d right?", name, age);
	if (age > 18)
	{
		printf("You can drink beer if you'd want to!");
	}
	else
	{
		printf("No beer for you!");
	}
	return 0;
}

File: test iphone.py
print("Hello there! I wrote this on an iPhone in Linux!")
name = input("Whats your name? ")
agestr = input("And your age is? ")
age = int(agestr)

print("Hello " + name + "!\nYour age is " + agestr + " right?")

if age > 18:
    print("You can drink beer if you'd want to!")
else:
    print("No beer for you!")
